### PR TITLE
HELLO MICROVERSE

### PR DESCRIPTION
**Pull request by Yulianav09**
# Hello Microverse Project
* Created a feature branch to make all the changes.
* Added a .gitignore file containing all the unnecessary files.

* Set up Linter for HTML and CSS
* Added a descriptive README file to the project using the Microverse template.
* Created an HTML file 
* Created a CSS file and linked it with the HTML
*  Created a Test file that is ignored

### CODE REVIEW

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: - [x] It is also important to use good PR title and summary when making a pull request attached below is an image of how your pr title and summary for this project should look like please make sure to make the correct changes before resubmitting for a new pull request. :+1: 
good-pr
- [x] Your summary in this pr is good but you need to edit your PR title to be more descriptive :+1:

File: README.md [hunk 1]
```diff
@@ -1,2 +1,175 @@
...
+- **Implemented Linters**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://github.com/Yulianav09/Hello_microverse/tree/feature/Hello_microverse)
```
Comment (Efiamotu-1) [inline README.md]: - if your project does not have a live demo, it is not a requirement to add it to your readme file. please you can remove this section :+1:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,175 @@
...
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Efiamotu-1) [inline README.md:157]: - please this particular link is broken as it currently does not lead to your license. i kindly suggest you follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly setup your license file :+1:

File: README.md [hunk 3]
```diff
@@ -1,2 +1,175 @@
...
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (Efiamotu-1) [inline README.md]: - you can also remove these text as it is mainly a description of how you are to customise this section

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Yulianav09 , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,2 +1,165 @@
...
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```
Comment (franclobo) [inline README.md]: - **[OPTIONAL]** When you have removed a section from the README please 🙏🏽 also remove it from the menu for professional documentation.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,165 @@
...
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
```
Comment (franclobo) [inline README.md]: - **[OPTIONAL]** When you have removed a section from the README please 🙏🏽 also remove it from the menu for professional documentation.

**Review by franclobo — APPROVED** [review]
Hi @Yulianav09, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**